R/build-logo.R

```diff
@@ -7,19 +7,49 @@ build_logo <- function(pkg = ".") {
 
   file_copy_to(pkg, logo_path, from_dir = path_dir(logo_path))
 
-  if (!requireNamespace("magick", quietly = TRUE)) {
-    message("magick not available, using default pkgdown favicon.ico")
+  cat_line("Creating ", dst_path("favicon.ico"))
```

Comment:
This line is outdated. What about `message("Creating favicon set")`? I don't think it would be useful to list all files.

Reply:
:+1: